cmd/podman/create.go

```diff
@@ -82,33 +92,27 @@ func createCmd(c *cli.Context) error {
 	return nil
 }
 
-func createInit(c *cli.Context) error {
+func createInit(c *cliconfig.PodmanCommand) error {
 	// TODO should allow user to create based off a directory on the host not just image
 	// Need CLI support for this
 
 	// Docker-compatibility: the "-h" flag for run/create is reserved for
 	// the hostname (see https://github.com/containers/libpod/issues/1367).
-	if c.Bool("help") {
```

Comment:
I take it that cobra CLI handles this properly?

If so, you can remove the docker-compat comment above as well

Reply:
presumably